src/js/actions/deploymentActions.js

```diff
@@ -60,7 +59,15 @@ export const getDeploymentsByStatus = (status, page = default_page, per_page = d
         }
         return accu;
       },
-      [dispatch({ type: DeploymentConstants[`RECEIVE_${status.toUpperCase()}_DEPLOYMENTS`], deployments, deploymentIds, status })]
+      [
+        dispatch({
+          type: DeploymentConstants[`RECEIVE_${status.toUpperCase()}_DEPLOYMENTS`],
+          deployments,
+          deploymentIds,
+          status,
+          total: Number(res.headers['x-total-count'])
```

Comment:
nitpicking: x-total-count should be a const

Reply:
Good point - I'll extract this to the api modules, but in a different PR ;)

Reply:
#689

Reply:
thanks. approved.